src/main/views/search.njk

```diff
@@ -70,12 +70,21 @@
 
     <script src="/assets/js/accessible-autocomplete.min.js"></script>
     <script>
-      const element = document.querySelector('#search-input-container')
+      const element = document.querySelector('#search-input-container');
       const source = {{ optionList | dump |safe }}
         accessibleAutocomplete({
           element: element,
           id: 'search-input',
-          source: source
-        })
+          source: source,
+          displayMenu: 'overlay'
+        });
+      history.pushState(null, null, location.href);
+      history.back();
+      history.forward();
```

Comment:
this seems extremely hacky? forcing them back and forward?

Reply:
This was the only solution I found that covered back, forward, and refresh cases.
Others only worked for back and refresh